ember-client/shared/src/main/scala/org/http4s/ember/client/internal/WebSocketHelpers.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.ember.client.internal
+
+import cats.MonadThrow
+import cats.data.NonEmptyList
+import cats.syntax.all._
+import org.http4s.Status
+import org.http4s._
+import org.http4s.crypto.Hash
+import org.http4s.crypto.HashAlgorithm
+import org.http4s.headers.Connection
+import org.http4s.headers._
+import org.http4s.websocket.Rfc6455
+import org.typelevel.ci._
+import scodec.bits.ByteVector
+
+private[client] object WebSocketHelpers {
+
+  private[internal] val supportedWebSocketVersion = 13L
```

Comment:
Adding `final` makes it a constant that will be inlined, because the right-hand-side is known at compile-time.

```suggestion
  private[internal] final val supportedWebSocketVersion = 13L
```